docs/ad/api.md

```diff
@@ -389,6 +550,176 @@ POST _opendistro/_anomaly_detection/detectors/results/_search
 }
 ```
 
+To find an ordered set of anomaly records for an entity with an anomaly within a certain time range for a specific feature value:
```

Comment:
From this description, the example below is to find anomaly of an `entity`, but the request query doesn't contain entity id filter.